cobra/flux_analysis/summary.py

```diff
@@ -129,23 +146,29 @@ def model_summary(model, threshold=1E-8, fva=None, floatfmt='.3g',
         raise NotImplementedError(
```

Comment:
I think can drop the `solver` , `solver_args` parameter and this exception altogether since we anyway introduce changes in behaviour